docs/guides/visualize-circuit-timing.ipynb

```diff
@@ -0,0 +1,7874 @@
+    "    included_channels=None,\n",
+    "    filter_readout_channels=False,\n",
+    "    filter_barriers=False,\n",
+    "    width=1000,\n",
+    ")\n",
+    "\n",
+    "# Display the figure\n",
+    "fig.show()\n",
+    "\n",
+    "# Save to a file\n",
+    "# fig.write_html(\"scheduler_timing.html\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "323a7d38",
+   "metadata": {},
+   "source": [
+    "#### Understand the generated figure\n",
+    "\n",
+    "The image of the circuit timing data output by `draw_circuit_schedule_timing` conveys the following information:\n",
+    "\n",
+    "- X axis is time in units of _dt_ seconds,  where 1 dt = 1 scheduling cycle.\n",
+    "- Y axis is the channel (think of channels as instruments that emit pulses).\n",
+    "    - `Receive channel` is the only channel that isn't an instrument by itself. It is an instruction played on all channels that are part of a communication procedure with the hub at that time.\n",
+    "    - `Qubit x` is the drive channel for qubit x.\n",
+    "    - `AWGRx_y` (arbitrary waveform generator readout) is used for readout channels communication for measuring qubits. The x and y arguments correspond to the readout instrument ID and the qubit number, respectively.\n",
+    "    - `Hub` controls broadcasting.\n",
+    "\n",
+    "Additionally, each instruction has the format of *X_Y*, where *X* is the name of the instruction and *Y* is the pulse type. A `play` type applies control pulses, and a `capture` records the qubit's state. You can also hover over each instruction to get more details. For example, the following figure shows a control pulse for the X gate applied to qubit 10 at 1161 dt:\n",
+    "\n",
+    "![Hovering over the output shows information such as the start, finish, and duration.](/docs/images/guides/visualize-circuit-timing/hover.svg 'Hovering over the figure for more details')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86af09ca",
+   "metadata": {},
+   "source": [
+    "### End-to-end example\n",
+    "\n",
+    "This example shows you how to enable the option, get it from the metadata, display it, and save it to a file.\n",
+    "\n",
+    "First, set up the environment, define the circuits and convert them to ISA circuits, and define and run the jobs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "b46908a0",
+   "metadata": {
+    "tags": [
+     "ignore-warnings"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "management.get:WARNING:2025-10-20 12:32:22,821: Loading default saved account\n",
+      "qiskit_runtime_service.__init__:WARNING:2025-10-20 12:32:31,261: Instance was not set at service instantiation. Free and trial plan instances will be prioritized. If you need a specific instance set it explicitly either by using a saved account with a saved default instance or passing it in directly to QiskitRuntimeService().\n"
```

Comment:
same as before, we probably don't want this output

Reply:
Yep.  I'm working to resolve that.  I could just manually delete it for now and put it in the bucket of notebooks to never run - but we're going to have this same output everywhere that we load the default saved account.  So we either need to a - come up with a way to hide cell output entirely, b - move all notebooks that load the default saved account into the "do not run" list, or c - change the code to log that information instead of output it.